baseapp/baseapp.go

```diff
@@ -221,6 +221,11 @@ func (app *BaseApp) LoadLatestVersion(baseKey *sdk.KVStoreKey) error {
 	return app.initFromMainStore(baseKey)
 }
 
+// WithRouter adds a new custom Router definition to the BaseApp
+func (app *BaseApp) WithRouter(rtr sdk.Router) {
```

Comment:
We should remove this as the base PR has `SetRouter` which checks if the baseapp is sealed or not.

Reply:
Removed